App.js

```diff
@@ -38,6 +38,7 @@ class Content extends React.Component {
         }
         this.handleLogin = this.handleLogin.bind(this);
         this.handleExit = this.handleExit.bind(this);
+        this.isLargeScreen = props.isLargeScreen;
```

Comment:
isLargeScreen sounds like should be a state for App.js instead of a global variable. Try to include it in state definition and get it managed dynamically by React native.

Reply:
Yeah I tried to do it dynamically, but it doesn't allow getting window dimensions in the render function. This is the implementation I saw in the official react-navigation example and documentation